index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">  
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <ul>
+                <li><label id="checkbox_toggle" class="hamburger">&#9776;</label></li>
```

Comment:
- [ ] as the previous reviewer requested, kindly change the hamburger button from `<label>` to `<button>` tag and please make sure to increase the size of it to make it match figma